exonum-java-binding-core/rust/integration_tests/benches/jni_cache.rs

```diff
@@ -41,15 +42,7 @@ const TX_EXEC_EXCEPTION_CLASS: &str =
 
 lazy_static! {
     pub static ref VM: Arc<JavaVM> = create_vm_for_benchmarks_with_fakes();
-    pub static ref EXECUTOR: MainExecutor = {
-        let ex = MainExecutor::new(VM.clone());
-        ex.with_attached(|env| {
-            jni_cache::init_cache(env);
-            Ok(())
-        })
-        .unwrap();
-        ex
-    };
+    pub static ref EXECUTOR: MainExecutor = create_executor_with_cache_initialized(VM.clone());
```

Comment:
How comes it works if there is no such method? 😕

Reply:
Oops, missed that. But as I said before, benches aren't compiled/executed within our current testing process.